### ISSUE TITLE
Currently failing tests

### ISSUE THREAD

**Issue by dianashk** [pelias/acceptance-tests#192]
```
### search
  ✘ regression [2] "{"text":"brooklyn, ny"}": score 3 out of 4
  diff:
    label
      expected: Brooklyn, Kings County, NY, USA
      actual:   Brooklyn, New York, NY, USA
  ✘ regression [7] "{"text":"billerica"}": score 1 out of 2
  diff:
    label
      expected: Billerica, Middlesex County, MA, USA
      actual:   Billerica, River Pines, MA, USA
  ✘ regression [8] "{"text":"billerica, ma"}": score 1 out of 2
  diff:
    label
      expected: Billerica, Middlesex County, MA, USA
      actual:   Billerica, River Pines, MA, USA
  ✘ regression [10] "{"text":"chelsea, new york"}": score 1 out of 2
  diff:
    label
      expected: Chelsea, New York County, NY, USA
      actual:   Chelsea, London, United Kingdom
  ✘ regression [11] "{"text":"soho, new york"}": score 2 out of 4
  diff:
    label
      expected: Soho, New York County, NY, USA
      actual:   Soho, Brooklyn, NY, USA
    label
      expected: SoHo, Manhattan, NY, USA
      actual:   Soho, Brooklyn, NY, USA
  ✘ regression [1425586777012:3] "{"text":"paris"}": score 1 out of 2
  diff:
    label
      expected: Paris, France
      actual:   Paris, Paris-4E-Arrondissement, France
  ✘ regression [1425586777012:6] "{"text":"london"}": score 3 out of 6
  diff:
    label
      expected: London, Greater London, United Kingdom
      actual:   London, City of Westminster, United Kingdom
    label
      expected: London, Middlesex, Canada
      actual:   London, City of Westminster, United Kingdom
    label
      expected: London, United Kingdom
      actual:   London, City of Westminster, United Kingdom

### San Francisco
  ✘ regression [5] "{"text":"san francisco"}": score 2 out of 3
  diff:
    region
      expected: California
      actual:   Atlántida

### quattroshapes popularity
  ✘ regression [1] "{"layers":"coarse","text":"chelsea"}": score 3 out of 4
  diff:
    label
      expected: Chelsea, New York County, NY, USA
      actual:   Chelsea, London, United Kingdom
  ✘ regression [3] "{"layers":"coarse","text":"ridgewood"}": score 3 out of 4
  diff:
    label
      expected: Ridgewood, Queens County, NY, USA
      actual:   Ridgewood, Bergen County, NJ, USA
  ✘ regression [1-1] "{"layers":"coarse","text":"chelsea, ny"}": score 1 out of 2
  diff:
    label
      expected: Chelsea, New York County, NY, USA
      actual:   Chelsea, London, United Kingdom
  ✘ regression [3-1] "{"layers":"coarse","text":"ridgewood, ny"}": score 1 out of 2
  diff:
    label
      expected: Ridgewood, Queens County, NY, USA
      actual:   Ridgewood, Bergen County, NJ, USA

### Brooklyn
  ✘ regression [3] "{"text":"brooklyn","layers":"locality,localadmin","boundary.country":"USA"}": score 2 out of 3
  diff:
    region
      expected: New York
      actual:   Iowa
  ✘ regression [7] "{"text":"brooklyn","layers":"locality,localadmin"}": score 2 out of 3
  diff:
    region
      expected: New York
      actual:   Victoria

### autocomplete admin areas
  ✘ regression [2] "{"text":"new york"}": score 7 out of 8
  diff:
    label
      expected: New York, USA
      actual:   New York, Brooklyn, NY, USA
  ✘ regression [7] "{"text":"wales"}": score 1 out of 2
  diff:
    label
      expected: Wales, United Kingdom
      actual:   Powys, United Kingdom
  ✘ regression [11] "{"text":"madrid"}": score 4 out of 5
  diff:
    layer
      expected: locality
      actual:   venue

### admin lookup
  ✘ regression [17] "{"text":"ringland","layers":"neighbourhood"}": score 5 out of 6
  diff:
    county
      expected: Newport City
      actual:
  ✘ regression [18] "{"text":"california","layers":"region"}": score 8 out of 9
  diff:
    source
      expected: whosonfirst
      actual:   geonames
  ✘ regression [5] "{"point.lat":51.515049,"point.lon":-0.093229,"layers":"coarse"}": score 2 out of 3
  diff:
    region
      expected: London
      actual:
  ✘ regression [8] "{"text":"crown heights","layers":"coarse"}": score 6 out of 8
  diff:
    label
      expected: Crown Heights, Kings County, NY, USA
      actual:   Crown Heights, Poughkeepsie, NY, USA
    county
      expected: Kings County
      actual:   Dutchess County

### address parsing
  ✘ regression [3] "{"text":"450 w 37th st, new york, ny 11232"}": score 13 out of 14
  diff:
    unexpected result found
  ✘ regression [4] "{"text":"starbucks 10010"}": score 10 out of 14
  diff:
    name
      expected: Starbucks
      actual:   100 East 22nd Street
    housenumber
      expected: 304
      actual:   100
    street
      expected: Park Avenue South
      actual:   East 22nd Street
    label
      expected: Starbucks, Manhattan, NY, USA
      actual:   100 East 22nd Street, Manhattan, NY, USA

### address matching
  ✘ regression [2] "{"text":"507 hackney rd"}": score 6 out of 7
  diff:
    name
      expected: Hackney Cycles
      actual:   507 Hackney Road
  ✘ regression [3] "{"text":"49 Kay Street"}": score 2 out of 3
  diff:
    unexpected result found
```

Aggregate test results
Pass: 129
Fail: 24
Placeholders: 0
Regressions: 25
Took 81119ms
Test success rate 86%

**Comment by dianashk** [comment]
Related issues filed:

*Chelsea, NY:* https://github.com/whosonfirst/whosonfirst-data/issues/226

**Comment by dianashk** [comment]
**Crown Heights, NY**
> not clear what's happening since the record from WOF is in the index
> as can be seen [here](http://pelias.github.io/compare/#/v1/place%3Fids=whosonfirst:neighbourhood:85867069) however search doesn't surface it. Popularity is set to `1` so maybe it's just not able to score high enough to surface above the other results.

**Comment by dianashk** [comment]
**London**
>https://github.com/whosonfirst/whosonfirst-data/issues/225

**Comment by dianashk** [comment]
### search_poi
  ✘ [searchpoi-1] "{"text":"Target Eureka CA"}": score 4 out of 5
  diff:
    priorityThresh is 5 but found at position 10
  ✘ [searchpoi-2] "{"text":"Target Eureka California"}": score 4 out of 5
  diff:
    name
      expected: Target
      actual:   Calvary Chapel Eureka

### search
  ✘ regression [10] "{"text":"chelsea, new york"}": score 1 out of 2
  diff:
    label
      expected: Chelsea, New York County, NY, USA
      actual:   Chelsea, London, United Kingdom
  ✘ regression [11] "{"text":"soho, new york"}": score 3 out of 4
  diff:
    label
      expected: Soho, New York County, NY, USA
      actual:   Soho, Brooklyn, NY, USA
  ✘ [18] "{"text":"newark airport"}": score 9 out of 11
  diff:
    name
      expected: Newark Liberty International Airport
      actual:   Days Hotel Newark Airport
    label
      expected: Newark Liberty International Airport, Newark, NJ, USA
      actual:   Days Hotel Newark Airport, Newark, NJ, USA
  ✘ [20] "{"text":"statue of liberty","boundary.country":"USA"}": score 4 out of 5
  diff:
    region
      expected: New York
      actual:   Iowa
  ✘ [22] "{"text":"Paris","focus.viewport.min_lat":"34.635221","focus.viewport.min_lon":"-90.823908","focus.viewport.max_lat":"38.004041","focus.viewport.max_lon":"-82.197798"}": score 2 out of 3
  diff:
    region
      expected: Tennessee
      actual:   Paris
  ✘ [24] "{"focus.point.lat":37.743618,"focus.point.lon":-122.426117,"text":"zoo"}": score 1 out of 3
  diff:
    label
      expected: San Francisco Zoo, San Francisco, CA, USA
      actual:   Zoo, Bundaberg Central, Australia
    name
      expected: San Francisco Zoo
      actual:   Zoo
  ✘ [1425586777012:0] "{"text":"union sq"}": score 5 out of 8
  diff:
    name
      expected: Union Square
      actual:   2 Union Square
    county
      expected: San Francisco County
      actual:   Alameda County
    label
      expected: Union Square, San Francisco County, CA, USA
      actual:   2 Union Square, Union City, CA, USA
  ✘ regression [1425586777012:3] "{"text":"paris"}": score 1 out of 2
  diff:
    label
      expected: Paris, France
      actual:   Paris, Paris-4E-Arrondissement, France
  ✘ regression [1425586777012:6] "{"text":"london"}": score 3 out of 6
  diff:
    label
      expected: London, Greater London, United Kingdom
      actual:   London, City of Westminster, United Kingdom
    label
      expected: London, Middlesex, Canada
      actual:   London, City of Westminster, United Kingdom
    label
      expected: London, United Kingdom
      actual:   London, City of Westminster, United Kingdom

### San Francisco
  ✘ regression [5] "{"text":"san francisco"}": score 2 out of 3
  diff:
    region
      expected: California
      actual:   Atlántida

### quattroshapes popularity
  ✘ regression [1] "{"layers":"coarse","text":"chelsea"}": score 3 out of 4
  diff:
    label
      expected: Chelsea, New York County, NY, USA
      actual:   Chelsea, London, United Kingdom
  ✘ regression [3] "{"layers":"coarse","text":"ridgewood"}": score 3 out of 4
  diff:
    label
      expected: Ridgewood, Queens County, NY, USA
      actual:   Ridgewood, Bergen County, NJ, USA
  ✘ regression [1-1] "{"layers":"coarse","text":"chelsea, ny"}": score 1 out of 2
  diff:
    label
      expected: Chelsea, New York County, NY, USA
      actual:   Chelsea, London, United Kingdom
  ✘ regression [3-1] "{"layers":"coarse","text":"ridgewood, ny"}": score 1 out of 2
  diff:
    label
      expected: Ridgewood, Queens County, NY, USA
      actual:   Ridgewood, Bergen County, NJ, USA

### poi address matching
  ✘ [1] "{"text":"30 West 26th st, Manhattan, NY"}": score 7 out of 9
  diff:
    name
      expected: Samsung Accelerator
      actual:   30 West 26th Street
    label
      expected: Samsung Accelerator, Manhattan, NY, USA
      actual:   30 West 26th Street, Manhattan, NY, USA

### landmarks
  ✘ [14] "{"text":"statue of liberty"}": score 1 out of 2
  diff:
    label
      expected: Statue of Liberty, Manhattan, NY, USA
      actual:   Replica Statue of Liberty, Oregon-Nashua, IL, USA

### labels
  ✘ regression [12] "{"text":"New Zealand"}": score 1 out of 2
  diff:
    label
      expected: New Zealand
      actual:   New Zealand, Kaikoura District

### Autocomplete Streets
  ✘ [4-1] "{"text":"451 Fog Hill"}": no results returned
  ✘ [4-2] "{"text":"451 Fog Hill rd"}": no results returned
  ✘ [4-3] "{"text":"451 Fog Hill Road"}": no results returned

### Autocomplete POI
  ✘ [1-1] "{"text":"Target Eureka CA"}": no results returned
  ✘ [1-2] "{"text":"Target Eureka California"}": no results returned
  ✘ [1-4] "{"text":"Target Eureka","focus.point.lat":"40.801944","focus.point.lon":"-124.163611"}": no results returned

### autocomplete focus
  ✘ [2] "{"focus.point.lat":40.744243,"focus.point.lon":-73.990342,"text":"union square"}": score 1 out of 2
  diff:
    label
      expected: Union Square, New York County, NY, USA
      actual:   5 Union Square, Manhattan, NY, USA
  ✘ [3] "{"focus.point.lat":37.769316,"focus.point.lon":-122.484223,"text":"union square"}": score 1 out of 2
  diff:
    label
      expected: Union Square, San Francisco County, CA, USA
      actual:   2 Union Square, Union City, CA, USA
  ✘ [24] "{"focus.point.lat":37.743618,"focus.point.lon":-122.426117,"text":"zoo"}": score 2 out of 3
  diff:
    priorityThresh is 1 but found at position 9
  ✘ [25] "{"focus.point.lat":40.744131,"focus.point.lon":-73.990424,"text":"1 madison ave"}": score 5 out of 8
  diff:
    localadmin
      expected: Manhattan
      actual:   Auburn
    locality
      expected: New York
      actual:   Auburn
    priorityThresh is 1 but found at position 4

### autocomplete admin areas
  ✘ regression [2] "{"text":"new york"}": score 2 out of 4
  diff:
    label
      expected: New York, NY, USA
      actual:   New York, Brooklyn, NY, USA
    label
      expected: New York City, Manhattan, NY, USA
      actual:   New York, Brooklyn, NY, USA
  ✘ regression [3] "{"text":"london"}": score 2 out of 4
  diff:
    label
      expected: London, Greater London, United Kingdom
      actual:   London, City of Westminster, United Kingdom
    label
      expected: London, Middlesex, Canada
      actual:   London, City of Westminster, United Kingdom
  ✘ [7] "{"text":"wales"}": score 1 out of 2
  diff:
    label
      expected: Wales, United Kingdom
      actual:   Powys, United Kingdom
  ✘ [1-2] "{"text":"brook"}": score 1 out of 2
  diff:
    label
      expected: Brooklyn, New York, NY, USA
      actual:   Brka, Tuzla, Bosnia and Herzegovina
  ✘ [1-4] "{"text":"brookly"}": score 1 out of 2
  diff:
    label
      expected: Brooklyn, New York, NY, USA
      actual:   Brka, Tuzla, Bosnia and Herzegovina

### admin lookup
  ✘ [19] "{"text":"5 West 4th Avenue Canada"}": score 6 out of 7
  diff:
    housenumber
      expected: 5
      actual:   2005
  ✘ regression [5] "{"point.lat":51.515049,"point.lon":-0.093229}": score 2 out of 3
  diff:
    region
      expected: London
      actual:   City and County of the City of London
  ✘ regression [7] "{"text":"mission, ca","layers":"coarse"}": score 7 out of 8
  diff:
    priorityThresh is 5 but found at position 7

### address parsing
  ✘ regression [3] "{"text":"450 w 37th st, new york, ny 11232"}": score 13 out of 14
  diff:
    unexpected result found

Aggregate test results
Pass: 146
Fail: 23
Placeholders: 0
Regressions: 15
Took 78292ms
Test success rate 92%

**Comment by orangejulius** [comment]
Ok, we're now down to just a few tests failing. They roughly fall into X categories

- [ ] **County is Brooklyn, should be Manhattan** Will be fixed with the next build thanks to whosonfirst/whosonfirst-data#228
- [ ] **Geonames countries have neighborhood values** fixed in pelias/wof-admin-lookup#37, which was just released in pelias-wof-admin-lookup-1.4.0 and will be in the next build
- [ ] **London (the city) contains a region called "The City of Westminister"** Tracked in whosonfirst/whosonfirst-data-225.
